net-misc/nat-traverse/nat-traverse-0.6.ebuild

```diff
@@ -10,7 +10,6 @@ SRC_URI="https://gitlab.com/iblech/${PN}/blob/master/${P}.tar.bz2"
 LICENSE="GPL-2"
```

Comment:
should be `GPL-2+`, see: https://gitlab.com/iblech/nat-traverse/-/blob/master/nat-traverse-0.6.pl?ref_type=heads#L6-L8